command_line/export_best.py

```diff
@@ -0,0 +1,173 @@
+        Initialise the exporter
+
+        :param params: The phil parameters
+        :param experiments: The experiment list
+        :param reflections: The reflection tables
+        """
+
+        # Check the input
+        if not experiments:
+            raise Sorry("BEST exporter requires an experiment list")
+        if not reflections:
+            raise Sorry("BEST exporter require a reflection table")
+
+        # Save the stuff
+        self.params = params
+        self.experiments = experiments
+        self.reflections = reflections
+
+    def export(self):
+        """
+        Export the files
+        """
+        from dials.util import best
+
+        experiment = self.experiments[0]
+        reflections = self.reflections[0]
+        partiality = reflections["partiality"]
+        sel = partiality >= self.params.best.min_partiality
+        logger.info(
+            "Selecting %s/%s reflections with partiality >= %s",
+            sel.count(True),
+            sel.size(),
+            self.params.best.min_partiality,
+        )
+        if sel.count(True) == 0:
+            raise Sorry(
+                "No reflections remaining after filtering for minimum partiality (min_partiality=%f)"
+                % (self.params.best.min_partiality)
+            )
+        reflections = reflections.select(sel)
+
+        imageset = experiment.imageset
+        prefix = self.params.best.prefix
+
+        best.write_background_file(
+            "%s.dat" % prefix, imageset, n_bins=self.params.best.n_bins
+        )
+        best.write_integrated_hkl(prefix, reflections)
+        best.write_par_file("%s.par" % prefix, experiment)
+
+
+if __name__ == "__main__":
+    from dials.util.options import (
+        OptionParser,
+        flatten_experiments,
+        flatten_reflections,
+    )
+    from dials.util.version import dials_version
+    from dials.util import log
+    from dials.util import Sorry
+
+    usage = "dials.export models.expt reflections.pickle [options]"
+
+    parser = OptionParser(
+        usage=usage,
+        read_experiments=True,
+        read_reflections=True,
+        phil=phil_scope,
+        epilog=help_message,
+    )
+
+    # Get the parameters
+    params, options = parser.parse_args(show_diff_phil=False)
+
+    # Configure the logging
+    log.config(info=params.output.log, debug=params.output.debug_log)
+
+    # Print the version number
+    logger.info(dials_version())
+
+    # Log the diff phil
+    diff_phil = parser.diff_phil.as_str()
+    if diff_phil != "":
+        logger.info("The following parameters have been modified:\n")
+        logger.info(diff_phil)
+
+    if not params.input.experiments and not params.input.reflections:
+        parser.print_help()
+        sys.exit()
+
+    # Get the experiments and reflections
+    experiments = flatten_experiments(params.input.experiments)
+    reflections = flatten_reflections(params.input.reflections)
+
+    # do auto intepreting of intensity choice:
```

Comment:
As above, I don't think `params.intensity` is used in the `BestExporter`.

Reply:
Also as above.